content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,529 @@
+
+[1]: https://aws.amazon.com/s3/pricing/
+{{% /site-region %}}
+
+### Create an S3 bucket and set up an IAM policy
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+{{% op-datadog-archives-s3-setup %}}
+
+3. Create an IAM user and attach the above policy to it. Create access credentials for the IAM user. Save these credentials as `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+
+{{% op-datadog-archives-s3-setup %}}
+
+3. [Create a service account][1] to use the policy you created above.
+
+[1]: https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html
+
+{{% /tab %}}
+{{% tab "APT-based Linux" %}}
+
+{{% op-datadog-archives-s3-setup %}}
+
+3. Create an IAM user and attach the above policy to it. Create access credentials for the IAM user. Save these credentials as `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+
+{{% op-datadog-archives-s3-setup %}}
+
+3. Create an IAM user and attach the policy above to it. Create access credentials for the IAM user. Save these credentials as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% /tab %}}
+{{% tab "Terraform (AWS)" %}}
+
+{{% op-datadog-archives-s3-setup %}}
+
+3. Attach the policy to the IAM Instance Profile that is created with Terraform, which you can find under the `iam-role-name` output.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Connect the S3 bucket to Datadog Log Archives
+
+You need to connect the S3 bucket you created earlier to Datadog Log Archives so that you can rehydrate the archives later on.
+
+1. Navigate to Datadog [Log Forwarding][5].
+1. Click **+ New Archive**.
+1. Enter a descriptive archive name.
+1. Add a query that filters out all logs going through log pipelines so that those logs do not go into this archive. For example, add the query `observability_pipelines_read_only_archive`, assuming that no logs going through the pipeline have that tag added.
+1. Select **AWS S3**.
+1. Select the AWS Account that your bucket is in.
+1. Enter the name of the S3 bucket.
+1. Optionally, enter a path.
+1. Check the confirmation statement.
+1. Optionally, add tags and define the maximum scan size for rehydration. See [Advanced settings][6] for more information.
+1. Click **Save**.
+
+See the [Log Archives documentation][7] for additional information.
+
+## Install the Observability Pipelines Worker
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+The Observability Pipelines Worker Docker image is published to Docker Hub [here][1].
+
+1. Download the [sample pipeline configuration file][2].
+
+2. Run the following command to start the Observability Pipelines Worker with Docker:
+
+    ```shell
+    docker run -i -e DD_API_KEY=<API_KEY> \
+      -e DD_OP_PIPELINE_ID=<PIPELINE_ID> \
+      -e DD_SITE=<SITE> \
+      -e AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID> \
+      -e AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY> \
+      -p 8282:8282 \
```

Comment:
Add to the command
```
DD_ARCHIVES_BUCKET=<AWS_BUCKET_NAME>  
DD_ARCHIVES_SERVICE_ACCOUNT=<BUCKET_AWS_REGION>
```

and on line 188, add the following info about these two values, as we do in [this screenshot](https://a.cl.ly/5zubknyq)

Reply:
updated